### DISCUSSION TITLE
flomo to Anki flashcard

### DISCUSSION THREAD

**Discussion by moyuguy** [flomoapp/3rd-party-tools#29] [分享创造]
期待个功能：部分值得记忆的内容，支持转到anki里，看有没有大佬能有方案？

**Comment by ivanweng2077** [comment]
同期待
不过目前可以尝试使用随机漫步来达成需求

**Comment by feifeijin** [comment]
mark，也是在想这个问题。
可以调用导出功能，然后再倒入anki即可。

**Comment by heliang12369** [comment]
实现并不难，主要是Flomo每14天才可以导出一回，而且每回导出是全量的，由于这些问题，要是开发起来会比较复杂

**Comment by draJiang** [comment]
可以抓包用 Python 获取数据，然后自由处理数据。

**Comment by heliang12369** [comment]
我现在采用的方式是用RPA软件，按照标签把Memo拿出来，然后再导入到Anki中

**Comment by xxhk-org** [comment]
[flomo2anki](https://xxhk.org/flomo2anki)
